api/controllers/commentsController.js

```diff
@@ -0,0 +1,9 @@
+const { expungeComment } = require("../models/commentsModel");
+
+exports.deleteComment = (req, res, next) => {
+    expungeComment(req.params.comment_id).then(() => {
+        res.status(204).send()
```

Comment:
If you dont' want to return anything you can use res.sendStatus(204). This will only send your status code